src/buttons/CreateScenarioButton/components/CreateScenarioDialog/CreateScenarioDialog.js

```diff
@@ -128,6 +133,8 @@ const CreateScenarioDialog = ({
   function createScenarioData() {
     const scenarioData = {
       name: scenarioName,
+      description: scenarioDescription || null,
```

Comment:
I choose` ||` because the basic state of `scenarioDescription` is an empty string and default value of description in scenario API side is `null`, so I wanted to avoid sending an empty string in the payload hence `||` operator :slightly_smiling_face: could it be a problem or I'm overreacting ? :laughing:

Reply:
Oh my bad, I was expecting `scenarioDescription` to always be `null` or `undefined` when empty.

If we can send `null` to erase an existing scenario description, then it should work :+1: